src/main.py

```diff
@@ -1,9 +1,11 @@
-from raspgierry_pi import RaspgierryPi
-
 def main(argv=[]):
     game = RaspgierryPi()
```

Comment:
```suggestion
    game = Raspgierry_pi()
```

Reply:
According to PEP8 classes in Python are CamelCase